Classes/Options/ModuleOptions.php

```diff
@@ -55,8 +55,11 @@ class ModuleOptions implements \TYPO3\CMS\Core\SingletonInterface {
      */
     public function initializeObject() {
         $configuration = $this->configurationManager->getConfiguration(ConfigurationManagerInterface::CONFIGURATION_TYPE_FULL_TYPOSCRIPT);
+        $settings = $configuration['plugin.']['tx_web2pdf.']['settings.'];
+        $viewSettings = $configuration['plugin.']['tx_web2pdf.']['view.'];
```

Comment:
This will result in PHP notices (sth. like `Undefined index: tx_web2pdf.`) when `$configuration['plugin.']['tx_web2pdf.']['settings.']` is not set. Using `isset` instead should be safe.